src/processor/errors.ts

```diff
@@ -0,0 +1,34 @@
+export class GetUserGraphError extends Error {
+    constructor(message: string) {
+      super(message);
+      this.name = "GetUserGraphError";
+    }
+}
+  
+export class ApplyActionsError extends Error {
+    constructor(message: string) {
+      super(message);
+      this.name = "ApplyActionsError";
+    }
+}
+  
+export class CapacityLowError extends Error {
+    constructor(message: string) {
+      super(message);
+      this.name = "CapacityLowError";
+    }
+}
+
+export class StaleHashError extends Error {
+    constructor(message: string) {
+      super(message);
+      this.name = "StaleHashError";
+    }
+}
+
+export class UnknownError extends Error {
+    constructor(message: string) {
+      super(message);
+      this.name = "UnknownError";
+    }
+}
```

Comment:
General error types we expect as possible failure points for reconnection service